include/rocksdb/cache.h

```diff
@@ -450,6 +450,19 @@ enum PrimaryCacheType {
   kCacheTypeMax,
 };
 
+enum TieredAdmissionPolicy {
```

Comment:
I believe `enum class` is preferred, though `PrimaryCacheType` is already a plain enum, so OK for nearby consistency